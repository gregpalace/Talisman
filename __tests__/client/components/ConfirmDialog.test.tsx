import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ConfirmDialog from '../../../src/client/components/ConfirmDialog';

describe('React unit tests', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<ConfirmDialog />);
  });

  describe('<ConfirmDialog />', () => {
    it('should render correctly', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
