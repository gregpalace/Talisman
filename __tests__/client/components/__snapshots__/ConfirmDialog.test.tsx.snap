// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React unit tests <ConfirmDialog /> should render correctly 1`] = `""`;
